<launch>

  <arg name="save_directory" default="$(env HOME)/datasets/processed/anchors"/>
  <arg name="create_new_anchors" default="false"/>
  <param name="use_sim_time" value="true" />

  <node pkg="localization" type="ssloc_multiple.py" name="self_localization" output="screen">
    <remap from="panorama_image" to="/theta_z1/photo/image/compressed" />
    <param name="send_unity_pose" value="false" />
    <param name="map_frame_id" value="world"/>  
    <param name="camera_frame_id" value="camera_frame"/>      
    <param name="save_directory" value="$(arg save_directory)" />
    <param name="create_new_anchors" value="$(arg create_new_anchors)" />

  </node>  

</launch>
